[2023-04-09 08:23:14,579] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: multi_attack_dag.metadata manual__2023-04-09T08:22:58.418918+00:00 [queued]>
[2023-04-09 08:23:14,638] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: multi_attack_dag.metadata manual__2023-04-09T08:22:58.418918+00:00 [queued]>
[2023-04-09 08:23:14,647] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-04-09 08:23:14,648] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2023-04-09 08:23:14,649] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-04-09 08:23:14,694] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): metadata> on 2023-04-09 08:22:58.418918+00:00
[2023-04-09 08:23:14,736] {standard_task_runner.py:52} INFO - Started process 4269 to run task
[2023-04-09 08:23:14,743] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'multi_attack_dag', 'metadata', 'manual__2023-04-09T08:22:58.418918+00:00', '--job-id', '4796', '--raw', '--subdir', 'DAGS_FOLDER/attack_dag.py', '--cfg-path', '/tmp/tmpkt6bvsrx', '--error-file', '/tmp/tmp1ei79dec']
[2023-04-09 08:23:14,756] {standard_task_runner.py:80} INFO - Job 4796: Subtask metadata
[2023-04-09 08:23:15,039] {task_command.py:369} INFO - Running <TaskInstance: multi_attack_dag.metadata manual__2023-04-09T08:22:58.418918+00:00 [running]> on host ff3d40af78f9
[2023-04-09 08:23:15,368] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=multi_attack_dag
AIRFLOW_CTX_TASK_ID=metadata
AIRFLOW_CTX_EXECUTION_DATE=2023-04-09T08:22:58.418918+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-09T08:22:58.418918+00:00
[2023-04-09 08:23:15,376] {transport.py:157} INFO - Attempting refresh to obtain initial access_token
[2023-04-09 08:23:15,383] {client.py:777} INFO - Refreshing access_token
[2023-04-09 08:23:16,208] {python.py:173} INFO - Done. Returned value was: None
[2023-04-09 08:23:16,379] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=multi_attack_dag, task_id=metadata, execution_date=20230409T082258, start_date=20230409T082314, end_date=20230409T082316
[2023-04-09 08:23:16,452] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-04-09 08:23:16,611] {local_task_job.py:273} INFO - 3 downstream tasks scheduled from follow-on schedule check
